src/cat-data-service/src/cli.rs

```diff
@@ -1,15 +1,16 @@
-use std::sync::Arc;
-
-use crate::{db::MockedDB, logger, service, settings::Settings};
+use crate::{logger, service, settings::Settings, state::State};
 use clap::Parser;
+use std::sync::Arc;
 use tracing::subscriber::SetGlobalDefaultError;
 
 #[derive(thiserror::Error, Debug)]
 pub enum Error {
     #[error(transparent)]
-    ServiceError(#[from] service::Error),
+    Service(#[from] service::Error),
     #[error(transparent)]
-    LoggerError(#[from] SetGlobalDefaultError),
+    Logger(#[from] SetGlobalDefaultError),
```

Comment:
But this error will be handled here https://github.com/input-output-hk/catalyst-core/blob/d2aabde031464526ab2fbaf09f1ef4a60ca81776/src/cat-data-service/src/main.rs#L11, and if something will goes wrong it will definitely panics with the appropriate error. Or you think I should explicitly panic right here ?

Reply:
I think we can just use `logger::init().unwrap()` - if it returns an error, we're going to panic anyways, just panicking here means can keep the error enum uncluttered for the error variants we care about

Reply:
fixed